practical-k-means/README.md

```diff
@@ -0,0 +1,38 @@
+# Practical K-Means Clustering in Python
+
+The Jupyter notebooks in this directory follow the code examples in Real Python's [Practical K-Means Clustering in Python](https://realpython.com/practical-k-means-clustering-python/) article. The article is structured such that there are two main sections with code. The first section works with synthetic data. The second section starts when the TCGA cancer gene expression dataset is introduced.
+
+## Getting Started
+
+Follow the instructions below to get you up and running with a Jupyter notebook and all the code from the article.
```

Comment:
I suggest we remove the word "you" from this sentence. It sounds a little awkward to me:

```suggestion
Follow the instructions below to get up and running with a Jupyter notebook and all the code from the article.
```